src/components/utilities.js

```diff
@@ -65,3 +65,16 @@ export const qs = {
   parse: path => queryString.parse(path, { ignoreQueryPrefix: true }),
   stringify: params => queryString.stringify(params, { encodeValuesOnly: true })
 };
+
+export const processErrors = ({ request, update }) => {
+  const processErrorsSet = ({ errors, timestamp }) => errors.map((error, index) => ({
+    message: error.title,
+    type: 'error',
+    id: `error-${timestamp}-${index}`
+  }));
+  const hasPutErrors = update.isRejected;
+  const putErrors = hasPutErrors ? processErrorsSet(update) : [];
+  const hasGetErrors = request.isRejected;
+  const getErrors = hasGetErrors ? processErrorsSet(request) : [];
+  return getErrors.concat(putErrors);
+};
```

Comment:
Nice! I think this eventually will be replaced with redux dispatching actions when an error occurs but that's a follow up ticket. 

I took a stab at trying to reduce this down but didn't really come up with much 😅 (mostly let > const) 

```js
export const processErrors = ({ request, update }) => {
  let processErrorsSet = ({ errors, timestamp }) => errors.map((error, index) => ({
    message: error.title,
    type: 'error',
    id: `error-${timestamp}-${index}`
  }));

  let hasErrors = update.isRejected || request.isRejected;
  let putErrors = hasErrors ? processErrorsSet(update) : [];
  let getErrors = hasErrors ? processErrorsSet(request) : [];

  return getErrors.concat(putErrors);
};
```

Reply:
Ah, you're right haha. That does look better. Thanks!